tests/nat/mcp/test_mcp_session_management.py

```
@@ -604,8 +610,8 @@ async def hanging_aenter(self):
         mock_client.__aexit__ = AsyncMock(return_value=None)
 
         with patch('nat.plugins.mcp.client_base.MCPStreamableHTTPClient', return_value=mock_client):
-            with pytest.raises(RuntimeError, match="Session client initialization timed out after 60.0s"):
-                await function_group._create_session_client(session_id)
+            with pytest.raises(RuntimeError, match="Session client initialization timed out after 2.0s"):
```

Ruff (0.14.3)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)